tbl/summary_top_2_invade.tex

```diff
@@ -0,0 +1,11 @@
+\begin{tabular}{lrrl}
+\toprule
+                                    Player &  Median $p_1$ &  Memory Depth & Stochastic \\
+\midrule
+        Meta Winner Memory One: 31 players &         0.587 &            \(\infty\) &       True \\
```

Comment:
I don't think MWMO is surprising: at worst it'll do as good as the best memory one strategy right? So actually this is a neat outcome I would argue as it's confirming that memory one strategies do do well in N=2. Which in turn is nice when they do poorly for larger populations.